### PR TITLE
Update Tech Page

### PR DESCRIPTION
**Pull request by aaronosher**
- Updated communications with pro/con lists and platform descriptions
- Extended web hosting explanations, updated some statements, and linked more services
- Added section on applications and ticket software
- Added section on hardware loaning solutions

### CODE REVIEW

**Comment by bahorn** [comment]
So far so good. Assume you are planning some more updates?

**Comment by aaronosher** [comment]
> Assume you are planning some more updates?
Yeah, working on a bit about URL Shorteners and what HackNotts does.

**Comment by aaronosher** [comment]
Didn't realise that review request on a draft pr actually requests it

File: docs/organise/before/tech.md [hunk 1]
```diff
@@ -1,19 +1,91 @@
...
+
+If you need a fully fledged virtual machine, both [Google Cloud](https://cloud.google.com/free) and [Amazon Web Services](https://aws.amazon.com/free/) offer free tiers for [Compute Engine](https://cloud.google.com/compute) and [Elastic Compute Cloud (EC2)](https://aws.amazon.com/ec2/).
+
+## Ticketing and Applications
+There are generally two ways to manage sign-ups to your hackathon: tickets and applications.
+
+For both options there are plenty of options for platforms, but here are some of the most common.
+
+### Tickets
+When picking a ticketing platform, it's important to factor in what you value most (e.g. customizability, ticket management, mailing list support etc.).
```
Comment (aaronosher) [inline docs/organise/before/tech.md]: Is there a discussion about this anywhere else in the Wiki?

**Comment by bahorn** [comment]
Just checking, should you have included the `.vscode/settings.json` in the repo?

**Comment by aaronosher** [comment]
> Just checking, should you have included the `.vscode/settings.json` in the repo?

IDK, if you're using the [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) it's whitelisting the word `shortlinks`

**Comment by bahorn** [comment]
Ah ok, that fine.

**Pull request merged**